crates/astria-bridge-withdrawer/src/main.rs

```diff
@@ -83,3 +146,44 @@ async fn main() -> ExitCode {
     info!("withdrawer stopped");
     ExitCode::SUCCESS
 }
+
+fn read_frost_key<P: AsRef<std::path::Path>>(
+    path: P,
+) -> astria_eyre::eyre::Result<PublicKeyPackage> {
+    let key_str =
+        std::fs::read_to_string(path).wrap_err("failed to read frost public key package")?;
+    serde_json::from_str::<PublicKeyPackage>(&key_str)
+        .wrap_err("failed to deserialize public key package")
+}
+
+async fn initialize_frost_participant_clients(
+    endpoints: Vec<String>,
+    public_key_package: &PublicKeyPackage,
+) -> eyre::Result<HashMap<Identifier, FrostParticipantServiceClient<tonic::transport::Channel>>> {
+    let mut participant_clients = HashMap::new();
+    for endpoint in endpoints {
+        let mut client = FrostParticipantServiceClient::connect(endpoint)
```

Comment:
Note that initializing these can happen lazily/non-async by ussing `connect_lazy`.